java/metabase/db/liquibase/MetabaseExecuteClojureMigration.java

```diff
@@ -0,0 +1,88 @@
+package liquibase.change.custom;
```

Comment:
I'm not very happy with writing this as a java class, because it means we have to compile it separately and we have to execute the command `clj -X:deps prep` -- which Cam get ridded of [a while ago](https://github.com/metabase/metabase/pull/20611)

I wonder if this could be a clojure file and use `gen-class` to generate a javaclass for this instead.

Reply:
take a look at how I did all the other Liquibase custom classes -- I did them as a `proxy` which means we don't need a separate AOT step. We definitely don't want a separate AOT step if we can avoid it.

Reply:
There must be some way to register a custom change class with Liquibase?

Reply:
it seems like we don't have a way to register CustomTaskChange classes.

There is a [tutorial](https://liquibase.jira.com/wiki/spaces/CONTRIB/pages/1998880/Change) about register customchange (SQL changes) tho.

WDYT about spinning a new repo for this and including them as a dependency?
This way we don't have to do this AOT step.